demo/tsconfig.json

```diff
@@ -23,6 +23,7 @@
         "baseUrl": ".",
         "paths": {
             "*": [
+                "./node_modules/tns-core-modules/*",
```

Comment:
I see what you mean. I'm just wondering if you webpack an app that doesn't use full paths (but has a `tsconfig.aot.json` file): what difference would it make if the plugin uses full paths or not? I'd assume the paths are resolved by webpack similarly as the app's code.